examples/ct/ctmapper/mapper/mapper.go

```diff
@@ -194,8 +199,12 @@ func (m *CTMapper) oneMapperRun(ctx context.Context) (bool, error) {
 	if metaAny, err = ptypes.MarshalAny(mapperMetadata); err != nil {
 		return false, fmt.Errorf("failed to marshal mapper metadata as 'any': err %v", err)
 	}
+	metaBytes, err := proto.Marshal(metaAny)
```

Comment:
Why the "double wrapping" (i.e. meta->any->[]bytes rather than just `proto.Marshal(meta)`)?

Reply:
proto.Marshal would work.  The double wrapping preserves a bit of type checking.